dsl/jpql/src/main/kotlin/com/linecorp/kotlinjdsl/dsl/jpql/Jpql.kt

```diff
@@ -516,6 +517,16 @@ open class Jpql : JpqlDsl {
         )
     }
 
+    /**
+     * Creates an expression that represents the local time.
+     *
+     * This is the same as ```LOCAL TIME```.
+     */
+    @SinceJdsl("3.0.0")
```

Comment:
This version is 3.4.0, not 3.0.0. Looks like it needs fixing
```suggestion
    @SinceJdsl("3.4.0")
```